### DISCUSSION TITLE
What to do for updates?

### DISCUSSION THREAD

**Discussion by switchyfoot** [wolveix/satisfactory-server#340] [General]
Hello, I just set up this server again. When my game client updates, do I need to do anything to update the server?

Everything is working great btw, thanks for all your hard work on this.

**Comment by wolveix** [comment]
Hey @switchyfoot! Appreciate the kind words :) When there's an update available for the game, you can just restart the Docker container. To check for container image updates (not for the game), you'd want to do `docker pull wolveix/satisfactory-server:latest` :)

**Reply by 0x20fearless** [reply]
I did this today because the game updated via steam, had to stop the container, pull the latest image, it pulled the latest data, restarted the container and it lost all the configs, I could reload via a game save, but we lost 35h's of manufacturing items

Is that issue related to me stopping the container to update the server?

**Reply by wolveix** [reply]
@0x20Fearless you likely haven't bound your directories correctly, unfortunately. Please check that your container config is binding the `/config` directory within the container to a real directory on your host

**Reply by 0x20fearless** [reply]
> @0x20Fearless you likely haven't bound your directories correctly, unfortunately. Please check that your container config is binding the `/config` directory within the container to a real directory on your host

Ah I see I do not:
```services:
  satisfactory-server:
    container_name: 'satisfactory-server'
    hostname: 'satisfactory-server'
    image: 'wolveix/satisfactory-server:latest'
    ports:
      - '7777:7777/udp'
      - '7777:7777/tcp'
    volumes:
      - './satisfactory-server:/config'
    environment:
      - MAXPLAYERS=4
      - PGID=1000
      - PUID=1000
      - ROOTLESS=false
      - STEAMBETA=false
    restart: unless-stopped
    deploy:
      resources:
        limits:
          memory: 8G
        reservations:
          memory: 4G```
